iothub/device/src/Transport/Amqp/AmqpUnit.cs

```diff
@@ -364,10 +409,7 @@ public async Task<AmqpIoTOutcome> DisposeMessageAsync(string lockToken, AmqpIoTD
                 await EnableEventReceiveAsync(timeout).ConfigureAwait(false);
                 disposeOutcome = await _eventReceivingLink.DisposeMessageAsync(lockToken, AmqpIoTResultAdapter.GetResult(disposeAction), timeout).ConfigureAwait(false);
             }
-            if (Logging.IsEnabled)
-            {
-                Logging.Exit(this, lockToken, $"{nameof(DisposeMessageAsync)}");
-            }
+            Logging.Exit(this, lockToken, $"{nameof(DisposeMessageAsync)}");
```

Comment:
Out of curiosity, why not just `nameof(DisposeMessageAsync)`? Why wrap it in a $""?

Reply:
No reason at all, `nameof()` will give the result as a string, so it will the same. 
My guess, this might have been copied over from some other interpolated string log statement and then modified.

Reply:
I'll remove the occurrences that I find